lib/tasks/events.rake

```diff
@@ -0,0 +1,187 @@
+require "date"
+
+namespace :events do
+  require_relative "events_fixtures"
+  users = EventsFixtures::USERS
+  asset_types = EventsFixtures::ASSET_TYPES
+  context_id = EventsFixtures::CONTEXT_ID
+
+  desc "Generate random events into canvas_events table"
+  task gen_rand: :environment do
+    main(users, asset_types, context_id)
+  end
+end
+
+=begin
+  Todo Later (nice to haves):
+    [] Show percent complete bar to user
+    [] Ask user for event type, or array of event types
+    [] Ask the user for context_id
+    [] Ask the user for a list of students
+    [] Give the user an option to pull student data from course id
+=end
+
+def main(users, asset_types, context_id)
+  prompt_user :delete_events
+  num_records = prompt_user :num_records
+  chunk_size = prompt_user :chunk_size
+  start_date = prompt_user :start_date
+  end_date = prompt_user :end_date
+
+  create_rand_event_records(
+    num_records,
+    users,
+    asset_types,
+    context_id,
+    chunk_size,
+    start_date,
+    end_date,
+  )
+end
+
+def prompt_user option
```

Comment:
I don't think it's necessary to change, but this could also be a class with methods. Then to call it you could do something like `Prompter.send(:num_records)`